TestCases/cont_adj_euler/oneram6/inv_ONERAM6.cfg

```diff
@@ -2,8 +2,8 @@
 %                                                                              %
 % SU2 configuration file                                                       %
 % Case description: ONERA M6 wing in inviscid, transonic flow                  %
-% Author: Thomas D. Economon	      	      	                        	       %
-% Institution: Stanford University				                                     %
+% Author: Thomas D. Economon                                                       %
```

Comment:
This is nitpicky, but it looks like the header got messed up/was already messed up.

Reply:
fixed it!